js/templates/pageNav.html

```diff
@@ -43,16 +43,27 @@
           <% } %>
           ></a>
           <nav class="navList listBox clrBr clrP shad1 foldDown js-navList js-navPopMenu">
-            <a class="listLink">Current Node <i class="ion-android-arrow-dropdown floR"></i> </a>
-            <hr class="unleaded">
-            <a class="listLink"><i class="ion-android-bulb"></i> Foo <span class="floR clrT2">Cltrl + ?</span> </a>
-            <a class="listLink"><i class="ion-android-cloud-circle"></i> Foo</a>
-            <hr>
-            <a href="#transactions" class="listLink"><i class="ion-log-out"></i> Purchases</a>
-            <a href="#transactions" class="listLink"><i class="ion-log-in"></i> Sales</a>
-            <a href="#transactions" class="listLink"><i class="ion-briefcase"></i> Cases</a>
-            <hr>
-            <a class="listLink js-navSettings"><i class="ion-toggle-filled"></i> Settings</a>
+            <div class="listGroup clrS clrBr">
+              <a class="listLink clrS clrBr">Current Node <i
+                class="ion-arrow-right-b floR"></i> </a>
+            </div>
+            <div class="listGroup clrS clrBr">
+              <a class="listLink">My Page<span class="floR clrT2">Cltrl + ?</span> </a>
```

Comment:
I'm not sure if floating is the best approach here. There was an issue in v1 where translations of the menu item text could get really long and when they break to multiple lines, the design doesn't look so nice:

![image](https://cloud.githubusercontent.com/assets/794517/17110280/13203dcc-5251-11e6-9758-f97ce3a46878.png)

Would it be better to a.) make it into a tabular layout so the link text and shortcut text are in separate columns b.) Make the width of the pop menu expandable where it can get wider if the content demands it (of course, along with a max-width so it doesn't get too wide)? The latter would limit, but not completely eliminate text breaking to multiple lines.